### PR TITLE
Format the errors for the IDE

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cciutils/checks.py [hunk 1]
```diff
@@ -18,24 +18,42 @@
 
 def error(checker, message, file=None, line=None, col=None, error_type="error"):
     """
-    Write a GitHub error or warn message
+    Write a GitHub error or warn message id the CI environment variable is true,
```
Comment (arnaud-morvan) [inline c2cciutils/checks.py:21]: Write an error or warn message formatted for GitHub if the CI environment variable is true else for IDE.

File: c2cciutils/checks.py [hunk 2]
```diff
@@ -18,24 +18,42 @@
...
+    GitHub: ::(error|warning) file=<file>,line=<line>,col=<col>:: <checker>: <message>
+    IDE: [(error|warning)] <file>:<line>:<col>: <checker>: <message>
 
     See: https://docs.github.com/en/free-pro-team@latest/actions/reference/ \
         workflow-commands-for-github-actions#setting-an-error-message
     """
     result = ""
     if file is not None:
-        result += "file={}".format(file)
+        if os.environ.get("CI", "false").lower() == "true":
```
Comment (arnaud-morvan) [inline c2cciutils/checks.py:32]: Could help to do:
```
ci = os.environ.get("CI", "false").lower() == "true"
```
And reuse it later

**Review by arnaud-morvan — APPROVED** [review]
Looks good, see comments

File: c2cciutils/checks.py (hunk 2 above)
Comment (sbrunner) [reply c2cciutils/checks.py:32]: More readable, tanks :-)

**Pull request merged**